client/app/domain/options/delete/domain-options-delete.html

```diff
@@ -0,0 +1,15 @@
+<div data-ng-controller="controllers.Domain.Options.Delete as ctrl">
+    <div data-wizard
+         data-wizard-confirm-button-text="'domain_tab_options_delete_confirm_button' | translate"
+         data-wizard-hide-cancel-button="loading"
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="deleteDomain"
+         data-wizard-title="('domain_tab_options_delete_'+ctrl.option.option+'_title') | translate">
+        <div data-wizard-step data-wizard-step-valid="!ctrl.loading">
+            <p data-ng-if="!ctrl.loading" data-translate="domain_tab_options_delete_confirm"></p>
+            <div class="text-center" data-ng-if="ctrl.loading">
+                <oui-spinner></oui-spinner>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
Could this be turned into a `oui-modal` instead ?

Reply:
Of course, yes. But all the modals in the domain section are now implemented this way. I think it is better to have a separate task to convert all of them to oui-modals at one shot. Please let me know your thoughts on this :)

Reply:
As said above, I think the sooner we improve our code the better, especially if it is new code

Reply:
Agreed. This is done :)